libcontainer/criu_linux.go

```diff
@@ -1,3 +1,5 @@
+//go:build !runc_nocr
```

Comment:
I guess you chose `cr` to keep it more agnostic, but perhaps `no_criu` would be clearer; wdyt?

Reply:
renamed